### PR TITLE
[#4664] fix(server): Add the missing annotations fo access control interfaces

### PR DESCRIPTION
**Pull request by jerqi**
### What changes were proposed in this pull request?
Add the missing annotations for access control interfaces.
Fix the document by the way.

### Why are the changes needed?

Fix: #4664

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
By hand.

### CODE REVIEW

File: server/src/main/java/org/apache/gravitino/server/web/rest/PermissionOperations.java [hunk 1]
```diff
@@ -40,8 +40,10 @@
 import org.apache.gravitino.lock.LockType;
 import org.apache.gravitino.lock.TreeLockUtils;
 import org.apache.gravitino.metrics.MetricNames;
+import org.apache.gravitino.server.authorization.NameBindings;
 import org.apache.gravitino.server.web.Utils;
 
+@NameBindings.AccessControlInterfaces
```
Comment (xunliu) [inline server/src/main/java/org/apache/gravitino/server/web/rest/PermissionOperations.java:46]: Please add comments to describe the Class `NameBindings`.
I don't know its purpose.

Comment (jerqi) [reply server/src/main/java/org/apache/gravitino/server/web/rest/PermissionOperations.java:46]: Added.

**Comment by xunliu** [comment]
@jerqi Please fix CI failed.

**Comment by jerqi** [comment]
> @jerqi Please fix CI failed.

Sorry, I fixed.

File: conf/gravitino.conf.template [hunk 2]
```diff
@@ -63,7 +63,7 @@ gravitino.catalog.cache.evictionIntervalMs = 3600000
 # Whether Gravitino enable authorization or not
 gravitino.authorization.enable = false
 # The admins of Gravitino service, multiple admins are spitted by comma.
-gravitino.authorization.serviceAdmin = anonymous
+gravitino.authorization.serviceAdmins= anonymous
```
Comment (jerryshao) [inline conf/gravitino.conf.template:66]: Please add a space after "xxxAdmins".

Comment (jerqi) [reply conf/gravitino.conf.template]: Added.

**Review by jerryshao — APPROVED** [review]
LGTM. @xunliu do you want to take another look?

**Review by xunliu — APPROVED** [review]
LGTM

**Pull request merged**